tests/eval_files/trio100_simple_autofix.py

```diff
@@ -0,0 +1,51 @@
+import trio
+
+# a
+# b
+with trio.move_on_after(10):  # error: 5, "trio", "move_on_after"
+    # c
+    # d
+    print(1)  # e
+    # f
+    # g
+    print(2)  # h
+    # i
+    # j
+    print(3)  # k
+    # l
+    # m
+# n
+
+with trio.move_on_after(10):  # error: 5, "trio", "move_on_after"
+    ...
+
+
+# a
+# b
+# fmt: off
+with trio.move_on_after(10): ...;...;... # error: 5, "trio", "move_on_after"
+# fmt: on
+# c
+# d
+
+# Doesn't autofix With's with multiple withitems
+with (
+    trio.move_on_after(10),  # error: 4, "trio", "move_on_after"
+    open("") as f,
+):
+    ...
+
+
+# extreme case I'm not gonna care about, i.e. one item in the with, but it's multiline.
+# Only these leading comments, and the last one, are kept, the rest are lost.
+with (  # a
+    # b
+    # c
+    trio.move_on_after(  # error: 4, "trio", "move_on_after"
+        # e
+        9999999999999999999999999999999999999999999999999999999  # f
+        # g
+    )  # h
+    # i
+):  # this comment is kept
+    ...
```

Comment:
I can maybe detect this and abort the autofix, rather than eating the comments. But I'm not sure if this happens ever? Maybe if you're passing a lot of parameters to one of fail_after/fail_at/move_on_after/CancelScope so it's gotten split across lines by Black.

Reply:
Fixed, comments are now preserved.